ui/desktop/app/styles/app.scss

```diff
@@ -674,6 +674,10 @@ $chevron-down: url("data:image/svg+xml;utf-8,<svg viewBox='0 0 24 24' fill='%231
     display: flex;
     align-items: center;
     gap: 1rem;
+
+    ~ * {
```

Comment:
None blocking comment: did we check this does not affect other elements? asking because it is a bold selector lol, just checking 😉

Reply:
Yup, it should only be the ones after the header. It's not a common selector but it basically does the same thing as if we did `.hds-container > :not(.header-and-icon)` but just a bit more succinct assuming the header/icon is the first element